### DISCUSSION TITLE
Using bootstrap data- attributes with webpack

### DISCUSSION THREAD

**Discussion by ko-vasilev** [twbs/bootstrap#36208] [Q&A]
Hello.
I'm having a hard time using Bootstrap data attributes when building with Webpack.
I'm not using the Bootstrap outside of Webpack so I don't use the `expose-loader` module for that.

My solution right now is to add `<script>` element referencing the bootstrap.js file directly on my page.

Does anyone know a proper way of using data attributes in my case?

**Comment by AmProsius** [comment]
I think you will have to enable the components manually like so: https://getbootstrap.com/docs/5.1/components/tooltips/#example-enable-tooltips-everywhere
